### PR TITLE
Add images for python3.9

### PR DESCRIPTION
**Pull request by WinnerOK**
This PR is inspired by #52 thus might help to resolve [tiangolo/uvicorn-gunicorn-fastapi-docker#61](https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker/issues/61).

Unlike mentioned PR, this should be able to pass current testing in CI.

**_Note:_** if you currently need these images, you can access them at DockerHub: links are available in [forked repo](https://github.com/WinnerOK/uvicorn-gunicorn-docker).

### CODE REVIEW

File: docker-images/python3.9-alpine3.12.dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+COPY ./app /app
+WORKDIR /app/
+
+ENV PYTHONPATH=/app
+
+EXPOSE 80
+
+# Run the start script, it will check for an /app/prestart.sh script (e.g. for migrations)
+# And then will start Gunicorn with Uvicorn
+CMD ["/start.sh"]
```
Comment (graue70) [inline docker-images/python3.9-alpine3.12.dockerfile:26]: Missing newline

File: docker-images/python3.9-slim.dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+COPY ./app /app
+WORKDIR /app/
+
+ENV PYTHONPATH=/app
+
+EXPOSE 80
+
+# Run the start script, it will check for an /app/prestart.sh script (e.g. for migrations)
+# And then will start Gunicorn with Uvicorn
+CMD ["/start.sh"]
```
Comment (graue70) [inline docker-images/python3.9-slim.dockerfile:29]: Missing newline

File: docker-images/python3.9.dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+COPY ./app /app
+WORKDIR /app/
+
+ENV PYTHONPATH=/app
+
+EXPOSE 80
+
+# Run the start script, it will check for an /app/prestart.sh script (e.g. for migrations)
+# And then will start Gunicorn with Uvicorn
+CMD ["/start.sh"]
```
Comment (graue70) [inline docker-images/python3.9.dockerfile:24]: Missing newline

File: docker-images/python3.9-alpine3.12.dockerfile (hunk 1 above)
Comment (WinnerOK) [reply docker-images/python3.9-alpine3.12.dockerfile]: Thank you, fixed

**Comment by cooperbenson-qz** [comment]
Hi! Was just wondering if there was any reason this hasn't been merged. I'm about to start a new project and would love if there was an official 3.9 release to work from 🙂

**Comment by JulianVolodia** [comment]
@cooperbenson-qz probably because [this](https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker/issues/61#issuecomment-798420313) comment

> Feel free to use the images until @tiangolo merge either #64 or #52 at uvicorn-gunicorn-docker repository.

**Comment by anton-petrov** [comment]
Check this https://hub.docker.com/repository/docker/antonapetrov/ until tiangolo released new version.
I have built updated versions (vulnerability scanning is on) for:
* uvicorn-gunicorn
* uvicorn-gunicorn-fastapi
* node-frontend

**Comment by tiangolo** [comment]
Thanks for the interest! This was handled in https://github.com/tiangolo/uvicorn-gunicorn-docker/pull/52, there are now Python 3.9 tags available. :tada: 

Given that, I'll close this PR one now.

**Pull request closed without merging**